packages/components/src/utils/get-element-height.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2022 by Swiss Post, Information Technology
+ */
+
+export function getElementHeight(el: HTMLElement): string;
+export function getElementHeight(el: HTMLElement, classWhenShown: string): string;
+export function getElementHeight(el: HTMLElement, classesWhenShown: string[]): string;
+export function getElementHeight(el: HTMLElement, classesWhenShown: any = []): string {
+  if (!Array.isArray(classesWhenShown)) classesWhenShown = [classesWhenShown];
+
+  const classesToAdd = classesWhenShown.filter(klass => !el.classList.contains(klass));
```

Comment:
Please do not use german property names.
With for example "cssClass" you are more specific and closer to the way it would be named in backend code like C#.
But you are free to use other names as well...
when it is as clear as here what you are iterating about, I would also absolutely agree with an abbreviation like "c".

```suggestion
  const classesToAdd = classesWhenShown.filter(c => !el.classList.contains(c));
```

Reply:
The term "klass" is quite conventional, it is at least used by [Angular ](https://github.com/angular/angular/search?q=klass)and [VueJS](https://github.com/vuejs/core/search?q=klass). But I guess we can agree on a name to line up in the future.

Reply:
![](https://user-images.githubusercontent.com/1659006/203511891-68027b32-ceef-437b-82b3-ec7cdef39905.gif)

Reply:
I never saw this in VueJS in the past 5 years. But if you both like to use it I'm fine with that.